ambry-router/src/test/java/com.github.ambry.router/DeleteManagerTest.java

```diff
@@ -199,7 +199,6 @@ public void testVariousServerErrorCode()
     HashMap<ServerErrorCode, RouterErrorCode> map = new HashMap<>();
     map.put(ServerErrorCode.Blob_Expired, RouterErrorCode.BlobExpired);
     map.put(ServerErrorCode.Blob_Not_Found, RouterErrorCode.BlobDoesNotExist);
-    map.put(ServerErrorCode.Partition_Unknown, RouterErrorCode.BlobDoesNotExist);
```

Comment:
Anything not in the map gets mapped by default to `UnexpectedInternalError`. @xiahome could you confirm?